.github/workflows/MarketplaceRelease.yml

```diff
@@ -0,0 +1,30 @@
+name: Marketplace Release
+
+on:
+    push:
+        tags:
+            - "*-web-v*"
```

Comment:
so if i understand correctly this action is only triggered on `*-web-v*` tag push right?

Reply:
Yeap